Doc/library/pathlib.rst

```diff
@@ -921,6 +921,19 @@ call fails (for example because the path doesn't exist):
    .. versionadded:: 3.5
 
 
+.. method:: Path.readlink()
+
+   Return the path to which the symbolic link points (as returned by
+   :func:`os.readlink`)::
+
+      >>> p = Path('foo')
```

Comment:
I think it would be better if we reused the example in the `Path.symlink_to()` documentation:

```py
>>> p = Path('mylink')
>>> p.symlink_to('setup.py')
>>> p.readlink()
PosixPath('setup.py')
```

We can also replace the `p.resolve()` example with `p.readlink()` at https://docs.python.org/3/library/pathlib.html#pathlib.Path.symlink_to.

Reply:
Could you also update the link at https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module ?